README.md

```diff
@@ -37,3 +37,9 @@ vendor/madewithlove/license-checker-php/bin/licensed licenses:allowed
 ```
 vendor/madewithlove/license-checker-php/bin/licensed licenses:check
 ```
+
+### Automatically generate configuration
+This command will automatically generate an `.allowed-licenses` configuration based on the currently used licenses.
+```
+vendor/madewithlove/license-checker-php/bin/licensed config:generate
```

Comment:
This binary will also live in vendor/bin/licensed, so that might make the README a bit easier to get (and also allows for moving the binary within this repo without breaking how users interact with it)